### PR TITLE
Add GUI (interviews) build for NEURON

### PR DESCRIPTION
**Pull request by pramodk**

### CODE REVIEW

**Comment by pramodk** [comment]
Please retest

**Review by matz-e — APPROVED** [review]
This needs a rebase on top of latest `develop`.

File: deploy/deploy.lib [hunk 1]
```diff
@@ -549,7 +549,6 @@ EOF
 
 generate_archive_modules() {
     local root="${1}"
-    set -x
```
Comment (matz-e) [inline deploy/deploy.lib]: Noooooo!

**Comment by pramodk** [comment]
This is no longer required as Interviews now internal part of NEURON and we will be used with internal variant rather than separate package (for now at least, to avoid dependency version management and so on).

**Pull request closed without merging**